lib/evmchain/evmchain.go

```diff
@@ -83,6 +83,54 @@ func All() []Metadata {
 	return resp
 }
 
+// Mainnets returns all mainnet evmchain metadatas ordered by chain ID.
+func Mainnets() []Metadata {
+	var resp []Metadata
+
+	mainnets := []uint64{IDEthereum, IDOmniMainnet, IDArbitrumOne, IDBase, IDOptimism}
```

Comment:
I'd avoid defaulting to Ethereum mainnet, it is very expensive. I think the test can just pick the chains it wants, no need for this exhausting default values. We don't want to build a super generic loadgen tool, we just want to test something specific, lets keep the scope small and solution direct to the point.